infra/release/release_note.md

```diff
@@ -1,28 +1,46 @@
-# Release Note 0.2.0
+# Release Note 0.3.0
 
 ## ONE-vscode
 
 ### ONE Explorer View
 
-- More commands are supported.
-  - [Open with Text Editor] command
-  - [Open Containing Folder] command
-  - [Collapse All] command
-- Improve [Create Config] command
+- Add hide-and-show button
+- Show more files
+  - memory trace files
+  - performance trace files
+  - circle log files
+  - extra backend model binary files
+- Open files with dedicated editor/viewers
+  - circle files - 'Circle Viewer'
+  - memory trace files - 'Mondrian Viewer'
+  - performance trace files - 'Json Tracer'
+  - circle log files - 'Text Editor'
 
 ### Toolchain View
 
-- Show Installable Toolchains on ToolchainView
-  - official release version
-  - nightly release version
-- Install Toolchain
-- Uninstall Toolchain
-- Show already-installed Toolchain
-- Support sudo command on Toolchain
+- TODO: Fill this part
+
+### Target Device View
+
+- Add Device list on local connection
+    - currently ubuntu 18.04 host PC and Tizen device added 
+- Show usable Executors under Device
```

Comment:
```suggestion
- Add devices on local connection
    - Supported devices
      - Local PC (Ubuntu 18.04)
      - Connected Tizen devices
- Show usable executors under each device
```
@struss I tried to rephrase, may I?

Reply:
Instead of a `Local PC`, how about `A simulator in a local PC`, likewise, `Tizen devices connected through SDB` or `SDB connected Tizen devices` instead of  `Connected Tizen devices` ? :)

Reply:
> Instead of a Local PC, how about A simulator in a local PC


IMO, as 'executor' has a concept of 'simulator' and 'device' only includes the meaning of 'a local PC', I think 'Local PC' fits here. And 'simulator' can be added under 'Show usable executors under each device'.

> Tizen devices connected through SDB or SDB connected Tizen devices instead of Connected Tizen devices ? :)

👍

Reply:
> > Instead of a Local PC, how about A simulator in a local PC
> 
> IMO, as 'executor' has a concept of 'simulator' and 'device' only includes the meaning of 'a local PC', I think 'Local PC' fits here. And maybe 'simulator' can be added under 'Show usable executors under each device'. (@struss Could you check here?)

I agree with @dayo09 opinion. as `Simulator` is a kind of Executor and `local PC` is a kind of Device. but Executor will be implemented in backend, so we cannot state this in this page, I think



> > Tizen devices connected through SDB or SDB connected Tizen devices instead of Connected Tizen devices ? :)
> 
> 👍

this looks nice. @dayo09 is it okay to change to this?

Reply:
I changed the suggestion

`Connected Tizen devices` -> `SDB-connected Tizen devices`
`Show usable executors under each device` -> `Show usable executors under each device (ex. simulators)`

Reply:
I'll apply the change for now. Let's revisit after fix release note.